src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/AssetLayer/AssetLayer.tsx

```diff
@@ -11,6 +11,7 @@ import {
 import L from 'leaflet'
 import { useMapInstance } from '@amsterdam/react-maps'
 import isEqual from 'lodash/isEqual'
+import './style.css'
```

Comment:
Isn't it possible to create styled components instead of including (global) css rules?

Reply:
I understand your point and I would have used styled components if that would work...